worker/task.go

```diff
@@ -55,6 +55,7 @@ func NewTaskRun(
 		Expires:  claim.taskClaim.Task.Expires,
 	}
 	ctxt, ctxtctl, err := runtime.NewTaskContext(tp, info)
+	ctxt.LogEntry = log
```

Comment:
This should probably be a sub-logger that has a property with the `taskId:<taskId>`